LayoutTests/svg/css/svg-write-resources-null-maskImage-crash-expected.html

```diff
@@ -1,5 +1,3 @@
-<svg>
-    <g style="mask-image: none, url()">
-    </g>
+<svg style="mask-image: none, url()">
```

Comment:
Tried removing <svg> would cause image diff failure but remove <p> works.

Reply:
You probably don't need the `style` attribute.

Reply:
Yes! You are right, updated patch to remove the style attribute also.

Reply:
hmmm looks Linux doesnt like it.

Reply:
I think Linux doesn't like the test file either, if you look at the image screenshots: https://ews-build.s3-us-west-2.amazonaws.com/WPE-WK2-Tests-EWS/63504139-30198/svg/css/svg-write-resources-null-maskImage-crash-diffs.html

The test file is incorrect XML for Linux. You might need something so it detects both the test file and the ref as HTML. Maybe `<!DOCTYPE html>` will suffice. Or wrapping with a `<div>` the test as you did for the ref. 

As long as it would still repro the original crash.

Reply:
Yeah compare to previous fail diff the wrapped <div> helps, updated the test file in latest patch (double checked that this test file reproduces crash without change.)